README.md

```diff
@@ -1,54 +1,155 @@
+
+# Usage
+<!-- UPDATE -->
+Following is an example how this project works with streamlit.
+
+<div align="center">
+  <a href="https://github.com/metakgp/PROJECT_NAME">
+    <img width="90%" alt="image" src="https://github.com/Ananya-m0140/Git_KOSS/assets/153496011/37b7a1cf-0956-4545-9f31-6c45cc08b978">
+    </a>
+  <a href="https://github.com/metakgp/PROJECT_NAME">
+    <img width="90%" alt="image" src="https://github.com/Ananya-m0140/Git_KOSS/assets/153496011/f08c7a0a-1fd3-4c11-ab13-92bc120fea7f">
+
+  </a>
+</div>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+# Maintainer(s)
+
+- [Arpit Bhardwaj](https://github.com/proffapt)
+
+<!-- MARKDOWN LINKS & IMAGES -->
+
+[contributors-shield]: https://img.shields.io/github/contributors/metakgp/kronos2.0.svg?style=for-the-badge
+[contributors-url]: https://github.com/metakgp/kronos2.0/graphs/contributors
+[forks-shield]: https://img.shields.io/github/forks/metakgp/kronos2.0.svg?style=for-the-badge
+[forks-url]: https://github.com/metakgp/kronos2.0/network/members
+[stars-shield]: https://img.shields.io/github/stars/metakgp/gyft.svg?style=for-the-badge
+[stars-url]: https://github.com/metakgp/kronos2.0/stargazers
+[issues-shield]: https://img.shields.io/github/issues/metakgp/gyft.svg?style=for-the-badge
+[issues-url]: https://github.com/metakgp/kronos2.0/issues
+[license-shield]: https://img.shields.io/github/license/metakgp/gyft.svg?style=for-the-badge
+[license-url]: https://github.com/metakgp/kronos2.0/blob/main/LICENSE
+[wiki-shield]: https://custom-icon-badges.demolab.com/badge/metakgp_wiki-grey?logo=metakgp_logo&style=for-the-badge
+[wiki-url]: https://wiki.metakgp.org
+
+# Contact
+
+<p>
+📫 Metakgp -
+<a href="https://slack.metakgp.org">
+  <img align="center" alt="Metakgp's slack invite" width="22px" src="https://raw.githubusercontent.com/edent/SuperTinyIcons/master/images/svg/slack.svg" />
+</a>
+<a href="mailto:metakgp@gmail.com">
+  <img align="center" alt="Metakgp's email " width="22px" src="https://raw.githubusercontent.com/edent/SuperTinyIcons/master/images/svg/gmail.svg" />
+</a>
+<a href="https://www.facebook.com/metakgp">
+  <img align="center" alt="metakgp's Facebook" width="22px" src="https://raw.githubusercontent.com/edent/SuperTinyIcons/master/images/svg/facebook.svg" />
+</a>
+<a href="https://www.linkedin.com/company/metakgp-org/">
+  <img align="center" alt="metakgp's LinkedIn" width="22px" src="https://raw.githubusercontent.com/edent/SuperTinyIcons/master/images/svg/linkedin.svg" />
+</a>
+<a href="https://twitter.com/metakgp">
+  <img align="center" alt="metakgp's Twitter " width="22px" src="https://raw.githubusercontent.com/edent/SuperTinyIcons/master/images/svg/twitter.svg" />
+</a>
+<a href="https://www.instagram.com/metakgp_/">
+  <img align="center" alt="metakgp's Instagram" width="22px" src="https://raw.githubusercontent.com/edent/SuperTinyIcons/master/images/svg/instagram.svg" />
+</a>
+</p>
+
+# Credits
```

Comment:
https://github.com/metakgp/README.md/blob/256ba6d22b41617eb1b4223d12815260b038903d/README.md?plain=1#L159

Change the section like this. Credit @spookbite.